splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/export/ExportOperationIT.java

```diff
@@ -536,9 +544,9 @@ private String buildExportSQL(String selectQuery, String exportPath, String comp
         if (useNativeSyntax) {
             StringBuilder sql = new StringBuilder();
             sql.append("EXPORT TO '").append(exportPath).append("'");
-            sql.append(" AS 'csv' ");
```

Comment:
but wouldn't this remove the tests for exporter checking the syntax internally (not the parser)?